store/pop_obs.go

```diff
@@ -127,18 +129,23 @@ func (s *store) btGetPopulations(ctx context.Context, in *pb.GetPopulationsReque
 	out *pb.GetPopulationsResponse) error {
 	dcids := in.GetDcids()
 
+	// Create the cache key suffix
 	keySuffix := "-" + in.GetPopulationType()
 	if len(in.GetPvs()) > 0 {
 		util.IterateSortPVs(in.GetPvs(), func(i int, p, v string) {
 			keySuffix += ("-" + p + "-" + v)
 		})
 	}
 
+	// Generate the list of all keys to query cache for
 	rowList := bigtable.RowList{}
 	for _, dcid := range dcids {
-		rowList = append(rowList, util.BtPopPrefix+dcid+keySuffix)
+		btKey := util.BtPopPrefix + dcid + keySuffix
+		log.Printf("GetPopulations: Querying cache with key \"%s\"", btKey)
```

Comment:
Maybe we want to avoid logs like this (in a potentially big for loop) so that we don't swarm the logs.

Reply:
Removed logging in dfd1793